### ISSUE TITLE
Windows do not support HiDPI scaling

### ISSUE THREAD

**Issue by gasparportik** [octgn/OCTGN#1880]
All Octgn window contents look blurry when viewed on a high dpi screen(for me it's 4k at 150% scaling).
<img width="610" alt="octgn-1 5xscale" src="https://user-images.githubusercontent.com/552166/57581991-2be1a480-74c8-11e9-8695-51f4b2df8e91.PNG">
It's even worse at 200% scaling(though I'm not sure who would use that):
![octgn-2xscale](https://user-images.githubusercontent.com/552166/57582000-4ae03680-74c8-11e9-8e6f-d2fb2625f33f.PNG)

I checked the code and it looks like the implicit scaling supported by WPF is explicitly disabled with this commit: f48cdf7562def0ae77dc6b397ea995d1830b5ea4 
It would be really easy to re-enable it, and even improve it to work with multiple displays having different dpis.
Question is: is there something in Octgn that breaks once the scaling is enabled?

**Comment by BenMatteson** [comment]
brief testing doesn't show anything obvious, maybe @kellyelton remembers why that was done in the first place?

**Comment by kellyelton** [comment]
Did some testing with this enabled and there were bugs with windows positioning. Rolling it back to standard dpi awarness for now (not per monitor dpi awareness).

tldr: All should be well if you have one monitor, or all your monitors are the same dpi
